var/spack/repos/builtin/packages/cabana/package.py

```diff
@@ -67,6 +68,13 @@ def cmake_args(self):
             '-DBUILD_SHARED_LIBS=%s' % (
                 'On' if '+shared'  in self.spec else 'Off')
         ]
+
+        # Enable Cabana submodules based on flags above
```

Comment:
```suggestion
        # Require Cabana submodules based on flags above
        if '+mpi' in self.spec:
            options.append('-DCabana_REQUIRE_MPI=ON')
        if '+arborx' in self.spec:
            options.append('-DCabana_REQUIRE_ARBORX=ON')
```

Reply:
This isn't necessary since we don't have as strict version requirements for these, but would provide more consistent behavior